draft/src/random_drafter.rs

```diff
@@ -0,0 +1,11 @@
+use crate::api::*;
+use rand::Rng;
+
+pub struct RandomDrafter {
+}
+
+impl Drafter for RandomDrafter {
+    fn pick(&self, pack: &Vec<Card>, cards_owned: &Vec<Card>, draft_info: &DraftInfo) -> u32 {
+        return rand::thread_rng().gen_range(0, pack.len() as u32);
```

Comment:
I think you want to store the thread_rng in the struct so you don't recreate it each time. Also the return can be omitted.

Reply:
The compiler complained when I omitted the return because it couldn't infer the appropriate numeric type, should I have type cast it instead?

Reply:
Now moot since I changed this return type.